docs/framework/data/adonet/sql/authentication-in-sql-server.md

```diff
@@ -15,7 +15,7 @@ SQL Server supports two authentication modes, Windows authentication mode and mi
   
  With Windows authentication, users are already logged onto Windows and do not have to log on separately to SQL Server. The following `SqlConnection.ConnectionString` specifies Windows authentication without requiring users to provide a user name or password.  
   
-```  
+```csharp  
 "Server=MSSQL1;Database=AdventureWorks;Integrated Security=true;  
```

Comment:
```suggestion
"Server=MSSQL1;Database=AdventureWorks;Integrated Security=true;"
```

Also I'm not convinced this should be "csharp" but I'm late to this party so maybe that's been decided already.